src/remote/impl/agency.service.ts

```diff
@@ -45,4 +46,26 @@ export class AgencyService implements IAgencyService {
         Logger.log(`Connecting to agent ${agentId}`);
         return await this.http.requestWithRetry(requestConnect);
     }
+
+    /**
+     * Makes a call to register an agent/wallet with the multitenant aca-py instance
+     * Note that this introduces some new terminology closer to what aca-py uses:
+     *   walletName instead of walletId
+     *   label instead of agentId
+     * The return object includes a connectionData arg
+     */
+    public async registerMultitenantAgent(walletName: string, walletKey: string, label: string): Promise<any> {
+        const request: AxiosRequestConfig = {
+            method: 'POST',
+            url: this.baseUrl + '/v2/multitenant',
+            data: {
+                walletName,
+                walletKey,
+                label
+            }
+        };
+        const res = await this.http.requestWithRetry(request);
+        Logger.log(`Registered multitenant agent ${label}`);
```

Comment:
just a comment and not asking anything be done or considered.  We now have debug() statement available on our logger.